src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,112 @@
+package duke;
+
+import javafx.application.Application;
+import javafx.scene.Scene;
+import javafx.scene.control.Button;
+import javafx.scene.control.ScrollPane;
+import javafx.scene.control.TextField;
+import javafx.scene.layout.AnchorPane;
+import javafx.scene.layout.VBox;
+import javafx.stage.Stage;
+
+import java.util.Scanner;
+
+/**
+ * <h1>Duke IP Project</h1>
+ *
+ * <p>A simple task managing chatbot. Functionalities include saving tasks, completing tasks, and </p>
+ */
+public class Duke extends Application {
+
+    private Button sendButton;
+    private TextField userInputTextField;
+    private ScrollPane scrollPane;
+    private VBox dialogueContainer;
+    private Scene scene;
+
+    /**
+     * Entry point of Duke. Simulates and main application loop.
+     * @param args
+     */
+    public static void main(String[] args) {
+        Ui duke = new Ui();
+        duke.start();
+        Scanner sc = new Scanner(System.in);
+
+        /* Initial setting of password */
+        while (!duke.checkAuth()) {
+            System.out.println("Please set a user password for auth.");
+            String p1 = sc.nextLine();
+            System.out.println("Please confirm password");
+            String p2 = sc.nextLine();
+            if (p1.trim().equals(p2.trim())) {
+                duke.setPw(p1.trim());
+            }
+        }
+
+        /* Main App Loop */
+        int x = 1;
```

Comment:
Consider naming x as a more obvious name (maybe returnValue?) just to make it clearer.

Reply:
Agreed, even though the scope is small it might be clearer to use a more descriptive variable name. Looking at your takeInputAndReturn method maybe x can be renamed to commandStatus?